### PR TITLE
[KOGITO-6297] - Add example in kogito-operator repo

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira issue : https://issues.redhat.com/browse/KOGITO-6297

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]

File: examples/app/user-friendly/process-quarkus-example.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    matchLabels:
+      app: process-quarkus-example
+  template:
+    metadata:
+      labels:
+        app: process-quarkus-example
+    spec:
+      containers:
+        - name: process-quarkus-example
+          image: quay.io/vajain/process-quarkus-example:1.0
```
Comment (radtriste) [inline examples/app/user-friendly/process-quarkus-example.yaml:26]: To follow other examples:

```suggestion
          image: quay.io/yournamespace/process-quarkus-example:1.0
```

File: examples/app/user-friendly/process-quarkus-example.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# 1. Create container image for kogito runtime application.
+#      Quarkus : https://quarkus.io/guides/container-image
+#      Spring-boot : https://spring.io/guides/gs/spring-boot-docker/
```
Comment (radtriste) [inline examples/app/user-friendly/process-quarkus-example.yaml:3]:
```suggestion
# 1. Create container image for kogito runtime application.
#      Example image name: quay.io/yournamespace/process-quarkus-example:1.0
#
#      Quarkus : https://quarkus.io/guides/container-image
#      Spring-boot : https://spring.io/guides/gs/spring-boot-docker/
```

**Review by radtriste — COMMENTED** [review]
small update for the image tag

**Review by ricardozanini — APPROVED** [review]

**Review by spolti — APPROVED** [review]
@vaibhavjainwiz nice.
Approving, it can be merged after Tristan's suggestions are resolved.

**Comment by vaibhavjainwiz** [comment]
> @vaibhavjainwiz can you please update the RELEASE_NOTES?

done

**Review by spolti — APPROVED** [review]

**Pull request merged**